pyroma/ratings.py

```diff
@@ -266,13 +268,39 @@ def test(self, data):
 
     def message(self):
         if self._major_version_specified:
-            return (
-                "The classifiers should specify what minor versions of "
-                "Python you support as well as what major version."
-            )
+            return "The classifiers should specify what minor versions of "\
+                   "Python you support as well as what major version."
         return "You should specify what Python versions you support."
 
 
+class PythonRequiresVersion(BaseTest):
+    def test(self, data):
+        self.weight = 100
+        python_requires = data.get("python_requires", None)
+        requires_python = data.get("requires_python", None)
+
+        # python_requires and requires_python are turned into 'Requires-Python' header
+        # https://packaging.python.org/en/latest/guides/dropping-older-python-versions/#specify-the-version-ranges-for-supported-python-distributions
+        # Some tools seems to report only one (eg: test_complete, test_markdown fail when 'requires_python' is not checked).
+        # So using whichever has a value provided
+        python_requires = python_requires or requires_python
```

Comment:
What is going on here is that some sources of data for Pyroma are reading the tool-specific source configuration of the build backend (e.g. `setup.cfg` or `setup.py` for Setuptools), which use one of several different keys (e.g. `python_requires` for Setuptools, or `requires-python` for PEP 621) whereas others are reading the output core metadata out of the `METADATA` file, which has the `Requires-Python` field.

However, instead of trying to handle all of that here, simply add a line to the [METADATA_MAP dictionary](https://github.com/regebro/pyroma/blob/606db052ef127e664dd52d59ff57c6828974bfed/pyroma/projectdata.py#L15) mapping `requires_python` to `python_requires` (this project uses the legacy Setuptools-specific keys for historical reasons instead of the standard PEP 621 keys or core metadata fields, as it was around long before PEP 517 or 621 existed), and elide all this code here:

```suggestion
        python_requires = data.get("python_requires", None)
```

Reply:
Got it. Thanks.

Reply:
Thanks for the explanation.